tracer/trace.go

```diff
@@ -64,7 +64,13 @@ func (tr *Tracer) newTrace(dir, exe string) (*trace, error) {
 	)
 
 	// Add breakpoint for workflow start
-	err = t.addFuncBreakpoint(t.fnPkg+"."+t.fn, nil)
+	var fnName string
+	if t.fnStruct != "" {
+		fnName = t.fnPkg + ".(*" + t.fnStruct + ")." + t.fn
```

Comment:
This makes an assumption that the function is on a pointer of the struct which may not necessarily be true?

Reply:
It does. And I do agree it’s not good design, I wanted to keep the simplification on the cli argument.
Having said that, we probably should take the assumption out and just take user input `.”+ t.fnStruct+”.`

Reply:
I think that's fine for now and with this tool. We can support a "WorkflowFuncExpr" later.

Reply:
Dealer's choice. This we should factor in later